tests/rgw/sanity_rgw.py

```diff
@@ -157,8 +160,15 @@ def run(ceph_cluster, **kw):
         remote_fp = rgw_node.remote_file(file_name=f_name, file_mode="w")
         remote_fp.write(yaml.dump(test_config, default_flow_style=False))
 
+    env_vars = config.get("env-vars", [])
+    cmd_env = ""
+    if env_vars:
+        for i in range(len(env_vars)):
+            cmd_env += str(env_vars[i]) + " "
+
```

Comment:
I think we can reduce this to
```
cmd_env = " ".join(env_vars)
```

Reply:
@psathyan We will miss space needed between each environment if we use join() method

Reply:
Sorry.. I missed whitespace while testing. Kindly ignore my previous comment